doc/apiv3-documentation.apib

```diff
@@ -2149,17 +2149,19 @@ Revisions are sets of updates to files in the context of repositories linked in
 |:----------------:| --------------------------------------------------------------------------------------------------| ------------- | ----------- | -------------------- |
 | self             | This revision                                                                                     | Revision      | not null    | READ                 |
 | project          | The project to which the revision belongs                                                         | Project       | not null    | READ                 |
-| author           | The user that added this revision, if the authorName was mapped to a user in OpenProject       | User          |             | READ                 |
+| author           | The user that added this revision, if the authorName was mapped to a user in OpenProject          | User          |             | READ                 |
+| showRevision     | A URL to the repository view (outside APIv3) showing this revision                                | String        | not null    |                      |
```

Comment:
Ah, I agree. What's with the 'supported operation'. I left that empty on purpose as there is no resource to operate on.